snafu/benchmarks/dnsperf/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM registry.centos.org/centos:8
```

Comment:
CentOS 8 hits its EOL in December. I know you're using the stream repo below, but I think it might be a good idea to use a different base image as support is kind of in the air.

Reply:
We can use centos stream when it's available. We can update the base image anytime.

Reply:
Holding Centos 8 for now, then.